st2common/st2common/models/db/execution.py

```diff
@@ -179,7 +179,7 @@ class ActionExecutionOutputDB(stormbase.StormFoundationDB):
     runner_ref = me.StringField(required=True)
     timestamp = ComplexDateTimeField(required=True, default=date_utils.get_datetime_utc_now)
     output_type = me.StringField(required=True, default='output')
-    delay = me.IntField(required=True, default=0)
+    delay = me.IntField()
```

Comment:
Any particular reason why you removed that? I guess required with a default value doesn't make much sense / difference.

Reply:
Making it consistent with other delay references in other collections. No other reason.